viz_scripts/mapping_dictionaries.ipynb

```diff
@@ -13,73 +12,29 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "improved-tribute",
    "metadata": {},
    "outputs": [],
    "source": [
     "df_pur= pd.read_csv(r'auxiliary_files/purpose_labels.csv')\n",
     "df_re = pd.read_csv(r'auxiliary_files/mode_labels.csv')\n",
-    "df_ei = pd.read_csv(r'auxiliary_files/energy_intensity.csv')\n",
+    "df_EI = pd.read_csv(r'auxiliary_files/energy_intensity_updated.csv')\n",
     "\n",
     "#dictionaries:\n",
-    "dic_pur = dict(zip(df_pur.purpose_confirm,df_pur.bin_purpose)) # bin purpose\n",
-    "dic_re  = dict(zip(df_re.replaced_mode,df_re.mode_clean)) # bin modes\n",
-    "dic_ei  = dict(zip(df_ei.replaced_mode,df_ei.EI)) # EI values"
+    "dic_pur = dict(zip(df_pur['purpose_confirm'],df_pur['bin_purpose'])) # bin purpose\n",
+    "dic_re  = dict(zip(df_re['replaced_mode'],df_re['mode_clean'])) # bin modes\n",
+    "dic_fuel = dict(zip(df_EI['mode'],df_EI['fuel']))"
```

Comment:
or this one

Reply:
I see that this is actually a column from `df_ei`, so it still works.

Reply:
The dic_ei is not needed here anymore, because I'm now reading that data suing the stored dataframe, then I use built the dictionary inside the energy intensity function.